libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -353,6 +358,7 @@ public void startNavigation(NavigationViewOptions options) {
     clearMarkers();
     // Initialize navigation with options from NavigationViewOptions
     if (!isInitialized) {
+      initLocale(options);
```

Comment:
Rather than store the `locale` and `unitType` here as class fields, we can pass them directly to the `InstructionView` and `SummaryBottomSheet` as indicated below.  

You're already doing the work https://github.com/mapbox/mapbox-navigation-android/pull/691/files#diff-efd7a5e1122379f7e999f95d086283fcR221 to init both of them for `NavigationViewModel` so that's covered 👍